### ISSUE TITLE
Include queries to produce the Batch / Historical graphs shown in documentation

### ISSUE THREAD

**Issue by laurikoobas** [databrickslabs/overwatch#287]
I just spent a very long day setting Overwatch up properly, ran through the bronze/silver/gold layers and ended up with some tables in the "overwatch" database. There's quite a few tables and some have very deep structures. I spent some time looking through these and have not figured out how to achieve for example the Heavy User Breakdown query/graph.

It would be very very nice to have a couple of sample queries at the end of the "aws_runner_docs_example_060" notebook to get started with some graphs.

**Comment by GeekSheikh** [comment]
You're not wrong there. It's near the top of our list to get some public, getting started queries released and added to the docs.

If you'll tell me what you're looking for exactly I'll post a quick query here for ya asap. But early next year we should have some getting started queries out in the docs for the public.

Appreciate the feedback. Glad to hear you were able to get it all deployed. :)

Happy Holidays!

**Comment by laurikoobas** [comment]
Thank you for actually replying! :)
The main focus would be to query/visualize something actionable, like commands/queries that took the longest to run.
For example I can see the `stage_runtime` field in the `sparkstage` table, but don't really see a way to link it to a query or a notebook or a user. To be honest I have the same problem when using Spark UI :)
